app.ts

```diff
@@ -0,0 +1,271 @@
+let fromParameter = 0;
+let timer: number;
+let recordCount: number;
+
+//// On Window Load
+
+window.onload = function () {
+  getRecordCount()
+    .then(() => {
+      return getHeadings();
+    })
+    .then(() => {
+      return getTable();
+    });
+  console.log("CALL HECTOR!!!");
+};
+
+const getParameters = (fromParameter: number): number => {
+  let toParameter: number;
+  let noOfRows = getNoOfRows();
+
+  toParameter = fromParameter + noOfRows;
+  return toParameter;
+};
+
+const getNoOfRows = (): number => {
+  const height = window.innerHeight;
+  let noOfRows = Math.floor(height / 40);
+  return noOfRows;
+};
+
+//// Functions To Create/Clear The HTML
+
+// Heading Row
+
+const createHeadingRow = (headingData: string) => {
+  const heading: any = document.querySelector("#heading");
+
+  let headings = `<div class="headings" id="headings">${headingData}</div>`;
+  heading.innerHTML += headings;
+};
+
+// Table Content
+
+const createTableContent = (contentData: string) => {
+  const content: any = document.querySelector("#content");
+
+  let table = `<div id="row-${contentData[0]}" class="rows"></div>`;
+  content.innerHTML += table;
+
+  let rows: any = document.querySelector("#row-" + contentData[0] + ".rows");
+  for (let x = 0; x < contentData.length; x++) {
+    let rowCols = `<div class="row_cols">${contentData[x]}</div>`;
+    rows.innerHTML += rowCols;
+  }
+};
+
+//// Fetch Requests
+
+// Heading Row (Getting the columns data)
+
+const getHeadings = () => {
+  fetch("http://localhost:2050/columns", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((res) => res.text())
```

Comment:
You should check for an error in `res` here. And you either want to remove the `()` around the `res` or specify the type, because currently the compiler thinks the type is `any`